tests/test_feature_aggregate.py

```diff
@@ -4,6 +4,13 @@
 
 from conftest import TEST_DTS
 
+
+def test_missing_region(check_aggregate_df):
+    print(check_aggregate_df['region'])
+    obs = check_aggregate_df.aggregate_region('Primary Energy', region='foo')
```

Comment:
what is the expected behavior if no subregions provided?

Reply:
It should agg all regions I think. @danielhuppmann ?